tools/pot/openvino/tools/pot/algorithms/quantization/optimization/params_tuning.py

```diff
@@ -39,14 +39,26 @@ def __init__(self, config, engine):
                         {'type': 'quantile', 'aggregator': 'mean', 'outlier_prob': 10e-4},
                         {'type': 'quantile', 'aggregator': 'mean', 'outlier_prob': 10e-5}
                     ]
+                },
+                'outputs': {
```

Comment:
@bonhunko, did you make sure that it works for the HW which does not need the outputs to be quantized, e.g. CPU or GPU.

Reply:
A unit test is required.

Reply:
Since the changes in params_tuning is not necessary with this PR, I removed this change.